spec/features/spree/admin/taxjar_settings_spec.rb

```diff
@@ -22,6 +22,20 @@
       click_on "TaxJar Settings"
     end
 
+    context "Taxjar reporting is enabled" do
+      it "shows that reporting is enabled" do
+        pending("Not implemented yet")
+
+        expect(page).to have_content("Transaction Sync")
+        expect(page).to have_select("Transaction Sync", selected: "No")
+        expect(page).to have_content("Sync orders and refund with TaxJar for automated sales tax reporting and filing. Complete and closed transactions sync automatically on update.")
```

Comment:
> "Complete and closed transactions sync automatically on update."

I'd probably roll with "Complete and closed transactions will automatically be synced when they are updated", else I could see the the "on update" part of the message telling someone "Your transactions are automatically backfilled on sync", which is not the case!

Reply:
Had a quick look at this and decided we may want to update the copy after speaking with Ryan. For now we keep the copy exactly the same as the Magento integration.